### PR TITLE
fix(security): use user id instead of session id in session page

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description

This PR fix the Logout User URL on Sessions page.

**Fixes** #4542

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 19.04.x
- [x] 19.10.x
- [x] 20.04.x
- [x] 20.10.x (master)

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/include/options/session/connected_user.php [hunk 1]
```diff
@@ -51,26 +51,26 @@
     FILTER_SANITIZE_STRING
 );
 
-$selectedUserSid = filter_var(
-    $_GET['session'] ?? null, // the sessionId of the chosen user
-    FILTER_SANITIZE_STRING
+$selectedUserId = filter_var(
+    $_GET['user'] ?? null, // the sessionId of the chosen user
```
Comment (sc979) [inline www/include/options/session/connected_user.php:55]:
```suggestion
    $_GET['user'] ?? null,
```

File: www/include/options/session/connected_user.php [hunk 2]
```diff
@@ -122,7 +122,7 @@
     if ($centreon->user->admin) {
         // adding the link to be able to kick the user
         $session_data[$cpt]["actions"] =
-            "<a href='./main.php?p=" . $p . "&o=k&session=" . $r['session_id'] . "'>" .
+            "<a href='./main.php?p=" . $p . "&o=k&user=" . $r['user_id'] . "'>" .
```
Comment (sc979) [inline www/include/options/session/connected_user.php:125]: Kindly, could you correct the line 142 too : 
https://github.com/centreon/centreon/pull/8876/files#diff-576736f130763acc5da4139d11faa7beR142

Comment (sc979) [reply www/include/options/session/connected_user.php:125]: but you may need to check the JS called

Comment (jeremyjaouen) [reply www/include/options/session/connected_user.php:125]: Fixed with #2386868

File: www/api/class/centreon_ldap_synchro.class.php [hunk 3]
```diff
@@ -78,16 +78,12 @@ public function postRequestLdapSynchro(): bool
...
-            $_POST['sessionId'] ?? null,
-            FILTER_SANITIZE_STRING
-        );
 
         if (!$this->isLdapEnabled()) {
             return $result;
         }
 
-        if (empty($contactId) && empty($sessionId)) {
+        if (empty($contactId)) {
```
Comment (sc979) [inline www/api/class/centreon_ldap_synchro.class.php:86]: You can remove the check on line 97 too :
https://github.com/centreon/centreon/pull/8876/commits/238686844d5e4385eaa05de87382aeb25202704f#diff-d7e73c17e00be39d953466d59c18aaa5R97

File: www/api/class/centreon_ldap_synchro.class.php [hunk 4]
```diff
@@ -93,15 +93,12 @@ public function postRequestLdapSynchro(): bool
...
-            // getting the contact name and ID for the logs
-            if ($contactId) {
-                // (getting the contactId to homogenize the next request's bindValue variable name)
-                $resUser = $this->pearDB->prepare(
-                    'SELECT `contact_id`, `contact_name` FROM `contact`
-                    WHERE `contact_id` = :contactId'
-                );
-                $resUser->bindValue(':contactId', $contactId, PDO::PARAM_INT);
-            }
+            // (getting the contactId to homogenize the next request's bindValue variable name)
```
Comment (sc979) [inline www/api/class/centreon_ldap_synchro.class.php:96]:
```suggestion
```

File: www/api/class/centreon_ldap_synchro.class.php [hunk 5]
```diff
@@ -75,15 +75,15 @@ public function postRequestLdapSynchro(): bool
...
 
         if (!$this->isLdapEnabled()) {
             return $result;
         }
 
-        if (empty($contactId)) {
+        if ($contactId === false) {
             $this->centreonLog->insertLog(
                 3, //ldap.log
                 "LDAP MANUAL SYNC : Error - Chosen contact data are missing."
```
Comment (sc979) [inline www/api/class/centreon_ldap_synchro.class.php:89]:
```suggestion
                "LDAP MANUAL SYNC : Error - Chosen contact id is not consistent."
```

**Pull request merged**